src/index.html

```diff
@@ -1,16 +1,60 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      class="form_box"
+      action="localhost"
+      method="post"
+      data-qa="big"
+    >
+      <label for="search_big">
+        <input
+          data-qa="keypress"
+          class="search search_big"
+          type="text"
+          id="search_big"
+          placeholder="Try &quot;Los Angeles&quot"
+        >
+      </label>
+    </form>
+
+
+
+    <form
+      class="form_box"
+      id="form"
+      data-qa="small"
+    >
+
+      <label for="search_small">
+        <input
+          class="search search_small"
+          type="text"
+          id="search_small"
+          placeholder="Try &quot;Los Angeles&quot"
+        >
+      </label>
+
+    </form>
   </body>
+
 </html>
```

Comment:
some of the spaces are just redundant
```suggestion
    <form
      class="form_box"
      id="form"
      data-qa="small"
    >
      <label for="search_small">
        <input
          class="search search_small"
          type="text"
          id="search_small"
          placeholder="Try &quot;Los Angeles&quot"
        >
      </label>
    </form>
  </body>
</html>
```